README.md

```diff
@@ -53,6 +53,85 @@ pip install -r requirements.txt -r requirements_dev.txt
 ```
 This script runs pytest on the `./tests` directory.
 
+## Debugging
+
+It is possible to debug this service for development by attaching a debugger
+that follows the
+[debugger-adapter-protocol](https://microsoft.github.io/debug-adapter-protocol/)
+to a harmony stack running in a local kubernetes cluster.  These instructions
+are for developers of this service in order to help them understand the code.
+
+Basic steps for debugging are:
+
+1. Add [debugpy](https://github.com/microsoft/debugpy) to `requirements.txt` file and reubild this image `./bin/build-image`.
+     - Generally this would just be adding a line to the end of the file that
+       says `debugpy==1.6.0` But there is a bug in that version and you need to
+       install from github directly until debugpy > 1.6 exists.  So for now,
+       you must install a specific version of `debugpy` and add `git` to the
+       service.Dockerfile
+
+     - docker/service.Dockerfile diff:
+
+         ```diff
+         -    && apt-get install -y python3-pip
+         +    && apt-get install -y python3-pip git
+         ```
+
+     -  requirements.txt diff:
+          ```diff
+          +git+https://github.com/microsoft/debugpy.git@b4cee28d7
+          ```
+
+1. Edit your harmony `.env` file to use
```

Comment:
Nit: All your numbered bullets are "1.", instead of incrementing.

Reply:
I was actually serious about my counterpoint though. Look at the markdown and you'll see them numbered 1 through N.

Reply:
Huh. I never knew that - that's actually super cool. Nice!